frontend/src/services/apiService.ts

```diff
@@ -68,6 +69,7 @@ export const api = {
     polls: polls,
     tags: tags,
     namespaces: namespaces,
+    users:users,
```

Comment:
```suggestion
    users: users,
```
I'm surprised Prettier lets this pass

Reply:
I think the CI is broken (again)